bigquery/docker-compose.yml

```diff
@@ -2,6 +2,11 @@ version: '3'
 services:
   elasticsearch:
     image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
+    environment:
```

Comment:
This configuration applies to all environments, including prod deployments. We don't want to disable bootstrap checks for prod deployments. Instead, can you do what `data-explorer` repo did?

https://github.com/DataBiosphere/data-explorer/blob/master/.circleci/config.yml#L53

Reply:
Oops, I realized this isn't used for prod deployment. Can you change the comment to this, to avoid future confusion?
```
# Disable bootstrap checks so users don't have to fiddle with vm.max_map_count.
# Note that this file is not used in production deployments; bootstrap
# checks are still enabled for production deployments.
```
I like this better than changing vm.max_map_count; I'll update the data-explorer repo.

Also, line 9 is not needed (I confirmed).

Reply:
Removed line 9

Reply:
Done